src/JSON/ClubData.json

```diff
@@ -116,8 +123,9 @@
     "tags": [
       "Community Service & Volunteer"
     ],
-    "description": "Students Offering Support (SOS) is a nation-wide charity that supports university chapters across Canada. In addition to general fundraising, we host Exam-Aid preparation services to fund SOS Global Impact Projects in developing countries. SOS is a unique social enterprise that relies on passionate student leaders to create a positive impact, both at home and abroad, through the SOS model of \u201cRaising Marks, Raising Money, Raising Roofs.\u201d",
-    "createdDate": "1920-01-15 14:38:49"
+    "description": "Students Offering Support (SOS) is a nation-wide charity that supports university chapters across Canada. In addition to general fundraising, we host Exam-Aid preparation services to fund SOS Global Impact Projects in developing countries. SOS is a unique social enterprise that relies on passionate student leaders to create a positive impact, both at home and abroad, through the SOS model of “Raising Marks, Raising Money, Raising Roofs.”",
+    "createdDate": "1920-01-15 14:38:49",
+    "slug": "uvic-students-offering-support-(sos)"
```

Comment:
Should probably do something about brackets too. Could remove the brackets and contents altogether or maybe just remove brackets.

Reply:
addressed in 387e914